src/renderer/components/ChatView.js

```diff
@@ -87,7 +91,7 @@ class ChatView extends React.Component {
     this.lastId = this.props.chat.id
     this.previousScrollHeightMinusTop = null
 
-    this.composerRef = React.createRef()
+    this.conversationRef = React.createRef()
```

Comment:
This breaks setting focus on the composer, used in `SplittedChatListAndView.js`.

Reply:
cc @Simon-Laux

Reply:
If that works, do it.